### PR TITLE
Please latest version of rubocop

### PR DESCRIPTION
**Pull request by lacostej**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  TargetRubyVersion: 1.9
+
+# not Ruby 1.9.3 compatible
```
Comment (ggilder) [inline .rubocop.yml]: What's the problem in 1.9.3?

File: .rubocop.yml [hunk 2]
```diff
@@ -1,5 +1,12 @@
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  TargetRubyVersion: 1.9
```
Comment (ggilder) [inline .rubocop.yml]: What does this do? Ideally Rubocop should be run under all the ruby versions tested in Travis.

Comment (lacostej) [inline .rubocop.yml]: > users may let RuboCop know the oldest version of Ruby which your project supports with:

From
https://github.com/bbatsov/rubocop/blob/master/manual/configuration.md#setting-the-target-ruby-version

Alternative is to use a .ruby-version

File: .rubocop.yml (hunk 1 above)
Comment (lacostej) [inline .rubocop.yml]: Current master fails on travis due to new cops added to rubocop.

Those cops enforce implementations that are not ruby 1.9 compatible (e.g. introduce %i[] for symbols from ruby 2.0)

For details: look at the history of this PR.

First attempt made the code rubocop compatible. It failed on ruby 1.9.3
https://travis-ci.org/commander-rb/commander/builds/267584518

Second attempt was some fiddlings that failed as well: https://travis-ci.org/commander-rb/commander/builds/267584518

Last attempt disabled the cops that are not Ruby 1.9 compatible. This passes the build everywhere. https://travis-ci.org/commander-rb/commander/builds/267591662

**Comment by ggilder** [comment]
Thanks for the explanation! LGTM.

**Pull request merged**